docs/UPDATING-DEPENDENCES.md

```diff
@@ -0,0 +1,64 @@
+# Updating dependencies
+
+We use open-source npm dependencies to build our application. These dependencies are (mostly) under active development and new versions are pushed fairly regularly.
+
+As many dependency updates fix small bugs or ensure compatibility with new platforms, we want to be keeping up with them. Letting dependencies fall behind several major versions is generally a bad scene.
+
+We run this process at least once every 2-week period (ie, once a sprint), or as soon as possible if we are alerted to a vulnerability. To know the last time the dependencies were updated, you can check [the git history of the `package.json`](https://github.com/cds-snc/cra-claim-tax-benefits/commits/master/package.json) file.
+
+## The update process
+
+1. Run `npm outdated`. This will list packages with newer versions than the ones we're using.
+
+2. Apply all updates — patch, minor, and major updates — by updating the version numbers in `package.json` to match the version in the `LATEST` column returned by `npm`.
+
+3. Delete the `package-lock.json` file and the `./node_modules` folder. This will ensure we get the latest nested dependencies as well.
+
+```
+rm -rf package-lock.json ./node_modules
+```
+
+4. Run the tests.
```

Comment:
I think we need to run an npm install to get the node_modules folder back before running the tests.